ui/app/templates/vault/cluster/secrets/backends.hbs

```diff
@@ -64,6 +64,9 @@
           <code class="has-text-grey is-size-8">
             {{backend.accessor}}
           </code>
+          <font class="has-text-grey is-size-8" style="margin-left:10px;">
```

Comment:
Getting linter errors that `style` on templates (hbs files) is not allowed, so we should instead use a class for the left margin (can add to `helpers.scss` and follow the pattern there)